auto_nag/scripts/common.py

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bugzilla.utils import get_config_path
+import json
+
+def get_login_info():
```

Comment:
Maybe we could that stuff in auto_nag.utils or move what is in utils here... as you wish